decrypt-api-key-in-action/nodejs/src/index.ts

```diff
@@ -0,0 +1,163 @@
+import { LitNodeClient, encryptString } from "@lit-protocol/lit-node-client";
+import { LitNetwork, LIT_RPC } from "@lit-protocol/constants";
+import {
+  createSiweMessage,
+  LitAbility,
+  LitAccessControlConditionResource,
+  LitActionResource,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+import { LitContracts } from "@lit-protocol/contracts-sdk";
+import { AccessControlConditions } from "@lit-protocol/types";
+import * as ethers from "ethers";
+
+import { litActionCode } from "./litAction";
+import { getEnv } from "./utils";
+
+const LIT_NETWORK = LitNetwork.DatilTest;
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+const LIT_CAPACITY_CREDIT_TOKEN_ID = process.env["LIT_CAPACITY_CREDIT_TOKEN_ID"];
+
+export const decryptApiKey = async (url: string, key: string) => {
```

Comment:
We should rename `url` to `alchemyUrl` since it's going to be concatenated with the API key and we can't guarantee it'll produce a valid URL if `url` is not an Alchemy partial URL